users/api/views.py

```diff
@@ -0,0 +1,56 @@
+"""
+.. See the NOTICE file distributed with this work for additional information
+   regarding copyright ownership.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+"""
+
+from django.contrib.auth import logout
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.decorators import api_view, permission_classes
+from .serializers import RegistrationSerializer
+
+
+@api_view(['POST'])
+def registration_view(request):
+    """
+    User registration endpoint, if the request is successful,
+    the response is formatted as a JSON object with a response message
```

Comment:
I can't see any code/annotation that makes this JSON, or any content-type control. Does it happen automatically this way?

Reply:
My bad, It's an `HttpResponse` not `JSON`